src/main/kotlin/com/cognifide/gradle/aem/pkg/deploy/CriticalInstallationError.kt

```diff
@@ -0,0 +1,22 @@
+package com.cognifide.gradle.aem.pkg.deploy
+
+enum class CriticalInstallationError(val className: String) {
+    CONSTRAINT_VIOLATION_EXCEPTION("javax.jcr.nodetype.ConstraintViolationException"),
+    DEPENDENCY_EXCEPTION("org.apache.jackrabbit.vault.packaging.DependencyException"),
+    SAX_EXCEPTION("org.xml.sax.SAXException");
```

Comment:
this should be available to be manually extended via `AemConfig.packageErrorExceptions` (?)